Remotion/ObjectBinding/Web.ClientScript/Scripts/BocCheckBox.ts

```diff
@@ -32,17 +32,17 @@ function BocCheckBox_InitializeGlobals (trueDescription, falseDescription)
 // Toggle the value of the checkbox.
 // checkBox: The check box.
 // label: The label containing the description for the value. null for no description.
-function BocCheckBox_ToggleCheckboxValue (checkBox, label, trueDescription, falseDescription)
+function BocCheckBox_ToggleCheckboxValue (checkBox: HTMLInputElement, label: HTMLLabelElement, trueDescription: Nullable<string>, falseDescription: Nullable<string>)
 {    
   checkBox.checked = !checkBox.checked;
   BocCheckBox_OnClick (checkBox, label, trueDescription, falseDescription);
 }
 
 //  Update the text-represention of the check-box value.
-function BocCheckBox_OnClick (checkBox, label, trueDescription, falseDescription)
+function BocCheckBox_OnClick (checkBox: HTMLInputElement, label: HTMLLabelElement, trueDescription: Nullable<string>, falseDescription: Nullable<string>)
 {    
  // Update the controls
-  var checkBoxToolTip;
+  var checkBoxToolTip; // TODO RM-7654: BocCheckBox_OnClick sets checkBoxToolTip but does not use it
```

Comment:
then we probably should delete this code. It's getting near it's retirement age anyway.